app/src/main/java/com/github/se/eventradar/viewmodel/EventsOverviewViewModel.kt

```diff
@@ -75,5 +76,12 @@ constructor(
 
 data class EventsOverviewUiState(
     val eventList: EventList = EventList(emptyList(), emptyList(), null),
+    var viewType: ViewType = ViewType.LIST,
+    var tab: Tab = Tab.BROWSE,
     val searchQuery: String = "",
 )
+
+enum class Tab {
```

Comment:
Good idea! Might be overkill for just two tabs but is a good way to make it scalable to more tabs.